csharp-basics/exercises/Arithmetic/Exercise 8/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace Exercise_8
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            double employee1 = TotalPay(7.5, 35);
+            double employee2 = TotalPay(8.2, 47);
+            double employee3 = TotalPay(10, 73);
+
+            if (employee1 > 0)
+            {
```

Comment:
Šim varēja atsevišķu validāciju izveidot, kur padod employee, jo 3x raksti vienu un to pašu